etw/rust/tracelogging/Cargo.toml

```diff
@@ -24,8 +24,10 @@ readme = "README.md"
 rust-version = "1.63"
 
 [features]
-default = ["etw", "macros"]
+default = ["macros", "etw", "user_mode"]
 etw = [] # Logging is enabled if windows && etw.
+user_mode = []
```

Comment:
We've not got a situation where people who explicitly enable the etw and macros features will no longer get events if they pick up this new crate version, unless they also modify their code to enable the user_mode feature.

It would probably be best to instead:
- Add only the kernel_mode feature, and assume if it is not enabled then we are user_mode
- Increment the crate version number

Reply:
* Removed user_mode feature
* Incremented crate version number